cli/src/test/java/hudson/cli/CLIConnectionFactoryTest.java

```diff
@@ -0,0 +1,33 @@
+package hudson.cli;
+
+import org.junit.Assert;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class CLIConnectionFactoryTest {
+
+    CLIConnectionFactory cliFactory;
+
+    @BeforeEach
+    void setUp() {
+        cliFactory = new CLIConnectionFactory();
+    }
+
+    @Test
+    void testBearerFromToken() {
+        Assert.assertEquals("Bearer some-token", cliFactory.of("some-token").authorization);
+    }
+
+    @Test
+    void testBearerFromEmptyToken() {
+        Assert.assertEquals("Bearer some-token", cliFactory.of("some-token").authorization);
```

Comment:
This is not empty when the test name suggests so?

Reply:
Sorry a missing cleanup.